sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableIO.java

```diff
@@ -208,6 +222,40 @@ public static Write write() {
     return Write.create();
   }
 
+  /**
+   * Creates an uninitialized {@link BigtableIO.ReadChangeStream}. Before use, the {@code
+   * ReadChangeStream} must be initialized with
+   *
+   * <ul>
+   *   <li>{@link BigtableIO.ReadChangeStream#withProjectId}
+   *   <li>{@link BigtableIO.ReadChangeStream#withInstanceId}
+   *   <li>{@link BigtableIO.ReadChangeStream#withTableId}
+   *   <li>{@link BigtableIO.ReadChangeStream#withAppProfileId}
+   * </ul>
+   *
+   * <p>And optionally with
+   *
+   * <ul>
+   *   <li>{@link BigtableIO.ReadChangeStream#withStartTime} which defaults to now.
+   *   <li>{@link BigtableIO.ReadChangeStream#withEndTime} which defaults to empty.
+   *   <li>{@link BigtableIO.ReadChangeStream#withHeartbeatDuration} with defaults to 1 seconds.
+   *   <li>{@link BigtableIO.ReadChangeStream#withMetadataTableProjectId} which defaults to value
+   *       from {@link BigtableIO.ReadChangeStream#withProjectId}
+   *   <li>{@link BigtableIO.ReadChangeStream#withMetadataTableInstanceId} which defaults to value
+   *       from {@link BigtableIO.ReadChangeStream#withInstanceId}
+   *   <li>{@link BigtableIO.ReadChangeStream#withMetadataTableTableId} which defaults to {@link
+   *       MetadataTableAdminDao#DEFAULT_METADATA_TABLE_NAME}
+   *   <li>{@link BigtableIO.ReadChangeStream#withMetadataTableAppProfileId} which defaults to value
+   *       from {@link BigtableIO.ReadChangeStream#withAppProfileId}
+   *   <li>{@link BigtableIO.ReadChangeStream#withChangeStreamName} which defaults to randomly
+   *       generated string.
+   * </ul>
+   */
+  @Experimental
```

Comment:
thanks for the good docs here. Do you think it makes sense to add some more documentation strings to the header of this file? e.g. links to how to start a changestream, the sort of data the transform outputs, the expected throughput, how to use with Beam schema (if supported), etc?

happy to leave for next PR, but I think it will be important.